src/renderer/containers/UploadWithTemplatePage/PageFooter/index.tsx

```diff
@@ -79,12 +78,16 @@ export default function PageFooter(props: Props) {
         )}
       </Button>
       <div className={styles.checkboxContainer}>
-        <Checkbox
+        <Switch
           checked={ShouldBeInLocal}
-          onChange={onCheckboxChange}
-        >
-          Store Locally
-        </Checkbox>
+          onChange={onSwitchChange}
+          checkedChildren="On"
+          unCheckedChildren="Off"
+        />
+        <span style={{ marginLeft: 8 }}>Only store on cloud</span>
```

Comment:
Generally this sort of css styling should go into the css file paired with the component (like on line 80 where it is assigned a className)